<?xml version="1.0"?>

<launch>
 
    <master auto="start"/>

    <!-- <include file="$(find human_body_detection)/launch/openni2.launch"/> -->

    <node
        pkg="human_body_detection"
        type="human_body_detection_node.py"
        name="human_body_detection"
        output="screen">
        <param name="models_pathname" value="/home/student1/openpose/models/"/>
        <param name="use_stereo" value="False"/>
        <param name="depth_neighborhood" value="10"/>
        <param name="accuracy_speed_ratio" value="2"/> <!-- 0 = accurate, 1 = optimal, 2 = fast -->
        <param name="topic_image_rgb_left" value="/camera/rgb/image_rect_color"/>
        <param name="topic_image_rgb_right" value=""/>
        <param name="topic_image_depth" value="/camera/depth_registered/hw_registered/image_rect_raw"/>
        <param name="topic_camera_info_left" value="/camera/rgb/camera_info"/>
        <param name="topic_camera_info_right" value=""/>
    </node>

</launch>

